fbfmaproom/fbf-update-data.py

```diff
@@ -262,6 +262,8 @@
         # TODO do this in Ingrid
         if 'Y' in ds and ds['Y'][0] > ds['Y'][1]:
             ds = ds.reindex(Y=ds['Y'][::-1])
+        if 'P' in ds:
+            ds = ds.chunk({'P': 1})
```

Comment:
Original chunking is a single chunk containing the entire variable. New chunking is one chunk per value of P (percentile).